src/util/index.ts

```diff
@@ -32,6 +39,8 @@ export {
   multiplyTransformMatrices,
   multiplyTransformMatrixArray,
   isIdentityMatrix,
+  angleToSkew,
+  skewToAngle,
```

Comment:
ae those useful?

Reply:
Do not know
But might be
Will save a dev frim reading about skew
Not important for me

Reply:
I removed them for now, when we have time to put a description over those and explain what they do and what is the value returned useful for, we can expose them.